cves/kernel/CVE-2019-9162.yml

```diff
@@ -251,8 +258,10 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    This vulnerability did not include anything related to internationalization.
```

Comment:
In what ways? Maybe explain how it doesn't have to do anything with foreign languages etc.

Reply:
Updated